client/asset/btc/btc.go

```diff
@@ -444,27 +463,22 @@ func BTCCloneWallet(cfg *BTCCloneCFG) (*ExchangeWallet, error) {
 	endpoint := btcCfg.RPCBind + "/wallet/" + cfg.WalletCFG.Settings["walletname"]
 	cfg.Logger.Infof("Setting up new %s wallet at %s.", cfg.Symbol, endpoint)
 
-	client, err := rpcclient.New(&rpcclient.ConnConfig{
-		HTTPPostMode: true,
-		DisableTLS:   true,
-		Host:         endpoint,
-		User:         btcCfg.RPCUser,
-		Pass:         btcCfg.RPCPass,
-	}, nil)
+	requester, err := newRequester(endpoint, btcCfg.RPCUser, btcCfg.RPCPass,
+		cfg.Symbol, cfg.Logger)
```

Comment:
I'd rather revert this for a couple of reasons. 1) the new function, `newRequester`, is only used once, and its name is not descriptive, since it actually returns an `rpcclient.Client`, not just a `RawRequester`. 2) It's a net increase in loc with what I feel is no improvement in readability.